tests/tests_pytorch/strategies/test_ddp_fully_sharded_native.py

```diff
@@ -140,18 +183,32 @@ def test_fully_sharded_native_strategy_checkpoint(tmpdir, precision):
 
 
 @RunIf(min_cuda_gpus=2, skip_windows=True, standalone=True, min_torch="1.12")
-def test_fully_sharded_native_strategy_checkpoint_multi_gpus(tmpdir):
+@pytest.mark.parametrize(
+    "model, strategy",
+    [
+        (TestFSDPModel(), "fsdp_native"),
+        (TestFSDPModelAutoWrapped(), DDPFullyShardedNativeStrategy),
```

Comment:
Since you are adding new parameterizations here, please make sure that the increase in test time is not unreasonable.

Reply:
it's not much locally. Just a few seconds.